### DISCUSSION TITLE
Nixie Shared Body - Spillway no longer triggers Return Valve

### DISCUSSION THREAD

**Discussion by orrious** [tagyoureit/nodejs-poolController#988] [Nixie] [answered]
Hey folks, hope all has been well.  I must have had an 'event' as I've had some equipment stop functioning.  As part of fixing everything, I upgrade to the latest versions of njsPC and REM.

I have a Nixie Shared Body with an Aux-Circuit "Spillway" of Type "Spillway".  Previously when the Spillway circuit was active, it would activate the "Return" Valve, which is connected to the valve that actuates the return.

In the latest version of njsPC, when the Spillway circuit is active, the Return valve is not enabled.

I have validated that the actuator, gpio, etc are all functional through REM.

**Comment by orrious** [comment, marked as answer]
It looks like logic must have changed.  The Spillway Circuit no longer appears to influence the Return Valve.  I originally had the Spillway actuator controlled by the Return Valve and the Spa Jets on their own Valve controlled by the Spa Circuit.  I've just swapped the Spillway and Spa Jets around and got it working again.

**Reply by tagyoureit** [reply]
Spillway should only divert the Return valve from Pool->Spa; the Intake should not be diverted.  It looks like my eyes got crossed and I accidentally commented something out recently.    Pull again and it should be back to normal.
